data/common.yaml

```diff
@@ -29,6 +29,24 @@ simp::scenario_map:
   poss:
     - pupmod
     - simp::scenario::poss
+
+  idm:
```

Comment:
You shouldn't need the following for this scenario (unless I'm missing something):

* issue
* ntpd
* simp_rsyslog
* simp_openldap::client
* simp::scenario::poss
* simp::pam_limits::max_logins
* useradd

You probably should include `pam` explicitly as well and I think we may need a ticket for breaking out our default lockdown settings or something. Not quite sure how to handle that one right now.